.github/workflows/build-test-distribute.yaml

```diff
@@ -13,6 +13,9 @@ permissions:
 env:
   KUMA_DIR: "."
   CI_TOOLS_DIR: "/home/runner/work/kuma/kuma/.ci_tools"
+concurrency:
+  group: ${{github.workflow}}-${{ github.ref == 'refs/heads/master' && github.sha || github.event.pull_request.number }}
```

Comment:
any reason not to just use `github.event_name == 'pull_request'`?

Reply:
i just mean why use `github.ref == 'refs/heads/master'`? don't we really mean `event_name == 'push'`? I find that makes it easier to understand what's happening

Reply:
you're right